lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -18,6 +18,7 @@ package OpenQA::WebAPI::Controller::API::V1::JobTemplate;
 use Mojo::Base 'Mojolicious::Controller';
 use Try::Tiny;
 use JSON::Validator;
+use Text::Diff;
```

Comment:
this is a new dependency hence you need to add it to `cpanfile` and the Dockerfile that spells out the requirement list explicitly as well and also in openQA.spec

Reply:
#2146